x/attribute/spec/README.md

```diff
@@ -1,3 +1,7 @@
+---
+sidebar_position: 1
+---
+
```

Comment:
What does this do?

Reply:
It will sort them in docusaurus, I'm having docusaurus autogenerate the sidebar for the SDK, this will ensure that the list of modules is first (it has position set to `0`) and that all the ones with the position set to `1` will come next in alphabetical order.

Reply:
I will test to see if it works with it commented, but with the change from the above conversation these will go away except in the list of modules. If the comment doesn't work I can possibly just append the comment in the script I use to pull it.